Dockerfile

```diff
@@ -1,14 +1,14 @@
-FROM amazonlinux:2018.03.0.20180827
+FROM amazonlinux:2.0.20200722.0
 
-ARG STACK_VERSION=2.3.1
-ENV \
-  PATH=/root/.local/bin:$PATH \
-  STACK_ROOT=/stack-root
+ARG STACK_VERSION=2.5.0.1
 
-RUN \
-  set -o xtrace && \
-  yum update -y && \
-  yum install -y \
+ENV LANG=C.UTF-8 \
+  LC_ALL=C.UTF-8 \
+  ITPROTV_ROOT=/home/itprotv \
```

Comment:
Do we really need this `ITPROTV_ROOT` stuff in here? Even `STACK_ROOT` should be unnecessary at this level.

Reply:
I guess that's true, I'm not sure what's best honestly. And right now I'm only really leaving it as is since I've already built the image 😝, but if I need to change it, I'll move it then, if not I'll move it once I'm finished